tools/lsp/wasm_main.rs

```diff
@@ -295,6 +295,29 @@ impl SlintServer {
                 let edit = crate::language::update_element(&self.ctx, position, properties);
                 let sn = self.ctx.server_notifier.clone();
 
+                if let Ok(edit) = edit {
+                    wasm_bindgen_futures::spawn_local(async move {
+                        let fut = sn.send_request::<lsp_types::request::ApplyWorkspaceEdit>(
+                            lsp_types::ApplyWorkspaceEditParams {
+                                label: Some("Element update".to_string()),
```

Comment:
For the remove (so not here), How about "Delete Element"? (this is what shows in the undo&redo stack)